config/config.php

```diff
@@ -27,6 +27,17 @@
 
     'secret' => env('JWT_SECRET'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | JWT Token Key Name
+    |--------------------------------------------------------------------------
+    |
+    | You can set a custom token key name in your .env file
+    |
+    */
+
+    'token_key_name' => env('JWT_TOKEN_KEY_NAME', 'token'),
```

Comment:
I don't think this needs to be configurable at the env level

Reply:
Maybe just `'token_key_name' => 'token',` should be better ?